pre_commit_hooks/pretty_format_json.py

```diff
@@ -55,6 +56,14 @@ def parse_topkeys(s):  # type: (str) -> List[str]
     return s.split(',')
 
 
+def get_diff(source, target):
+    source_lines = ''.join(source).split('\n')
+    target_lines = ''.join(target).split('\n')
```

Comment:
shouldn't need `''.join(...)` here, also you'll want to use `.splitlines(True)` and then `''.join(...)` later